### DISCUSSION TITLE
Shift Brightness Curve for Each Display

### DISCUSSION THREAD

**Discussion by dhrchan** [MonitorControl/MonitorControl#977] [Ideas]
Hi, 

The issue I'm having is that the maximum brightness on my MacBook is higher than the maximum brightness on my external display. I've tuned the "scale mapping curve" and "combined dimming switchover point" to sync the brightness between my MacBook and external display, but it is impossible to match the brightness at higher brightness settings due to the difference in maximum brightness for each display. 

I would like to be able to shift the brightness curve for my external display. For example, if I shift the brightness curve of my external display by + 10%, it will be at maximum brightness when the brightness slider is at 90%, and it will be at 10% brightness when the brightness slider is at 0%, and so on. 
![brightness](https://user-images.githubusercontent.com/22087718/158439190-072d0403-57c1-4483-a792-1e3bd28ad73a.png)

That way, I can use a  combined brightness slider for all displays, and the brightness of all my monitors will be synced. (Obviously, the external display won't be able to match the maximum brightness of my MacBook, but it will at least be at its maximum brightness.)

**Comment by waydabber** [comment]
This is a fair request, I created an issue about this one. https://github.com/MonitorControl/MonitorControl/issues/978. This is related to advanced absolute syncing (with calibration involved) in general. There are some other items that should go under this umbrella - see https://github.com/MonitorControl/MonitorControl/issues/943 fro example.

**Comment by steo-ml** [comment]
@dhrchan Not sure if I'm too late to the discussion, but if I understood your idea correctly, `DDC min` and `DDC max` in advanced settings might be able to achieve what you want. For example, setting 10 and 110 means that when the slider is at 0%, your external monitor would be at 10%, and when the reaches 90%, the monitor would be at 100%. If the slider goes higher, the monitor would just stay at 100% as it can't go any higher.

![image](https://user-images.githubusercontent.com/80666855/174516822-4bad4121-a5f1-4750-acf6-477098456daf.png)

**Reply by waydabber** [reply]
I'd leave DDC min at 0, but this might work, provided that the display has no issues receiving DDC values beyond 100 and interprets them simply as max brightness. This is firmware specific.
